flathunter/crawl_immobilienscout.py

```diff
@@ -5,15 +5,35 @@
 
 from flathunter.abstract_crawler import Crawler
 
+
 class CrawlImmobilienscout(Crawler):
     """Implementation of Crawler interface for ImmobilienScout"""
 
     __log__ = logging.getLogger('flathunt')
     URL_PATTERN = re.compile(r'https://www\.immobilienscout24\.de')
     RESULT_LIMIT = 50
 
-    def __init__(self):
+    def __init__(self, config):
         logging.getLogger("requests").setLevel(logging.WARNING)
+        self.config = config
+        self.driver = None
+        self.captcha_api_key = None
+        self.checkbox = None
+        self.afterlogin_string = None
+        if "captcha" in config:
+            self.captcha_api_key = self.config.get('captcha', dict()).get('api_key', '')
```

Comment:
All these lines have the same `self.config.get('captcha', dict())` construct. I think it makes sense to use a local variable here for that value.